ProcessLib/Utils/InitShapeMatrices.h

```diff
@@ -10,6 +10,7 @@
 #define PROCESSLIB_UTILS_INIT_SHAPE_MATRICES_H_
 
 #include <vector>
+#include <Eigen/StdVector>
```

Comment:
That might be obsolete now, cf. http://eigen.tuxfamily.org/bz/show_bug.cgi?id=829
Maybe this include even causes some problems. Could you check that, please?